src/tbp/monty/frameworks/experiments/profile.py

```diff
@@ -64,7 +64,7 @@ def __init_subclass__(cls, **kwargs):
         super().__init_subclass__(**kwargs)
         if cls.__bases__[0] is not ProfileExperimentMixin:
             raise TypeError("ProfileExperimentMixin must be leftmost base class.")
-        if not any([issubclass(b, MontyExperiment) for b in cls.__bases__]):
+        if not any(issubclass(b, MontyExperiment) for b in cls.__bases__):
```

Comment:
I totally forgot about this when I wrote this code. The list brackets aren't needed because the `x for x in items` bit is actually making a [generator](https://docs.python.org/3.8/reference/expressions.html#generator-expressions). In a single argument function that expects an iterable, that just works. So we don't have to create a list.